io/OGRWriter.cpp

```diff
@@ -154,11 +154,13 @@ bool OGRWriter::processOne(PointRef& point)
     double x = point.getFieldAs<double>(Dimension::Id::X);
     double y = point.getFieldAs<double>(Dimension::Id::Y);
     double z = point.getFieldAs<double>(Dimension::Id::Z);
-    double m = point.getFieldAs<double>(m_measureDim);
 
     OGRPoint pt(x, y, z);
     if (m_measureDim != Dimension::Id::Unknown)
+    {
```

Comment:
Not that it matters, but I think this change can go away. If `m_measureDim` is Unknown, `getFieldAs` will return 0, which is how OGRPoint initializes the measure dimension. You might test this, but I'm reasonably certain.

Reply:
I agree with that.